React/Views/RCTView.m

```diff
@@ -1169,6 +1172,17 @@ - (void)displayLayer:(CALayer *)layer
   backgroundColor = _backgroundColor.CGColor;
 #endif
 
+  #if TARGET_OS_OSX // [TODO(macOS ISS#2323203)
+  CATransform3D transform = self.transform3D;
+  if (layer.anchorPoint.x == 0 && layer.anchorPoint.y == 0 && !CATransform3DEqualToTransform(transform, CATransform3DIdentity)) {
+    // This compensates for the fact that layer.anchorPoint is {0, 0} instead of {0.5, 0.5} on macOS for some reason.
```

Comment:
seems like we should add https://developer.apple.com/documentation/quartzcore/calayer/1410817-anchorpoint in  comment?

Reply:
Sure. It's worth noting that there are a lot of layer properties like this one we _might_ want to special case for macOS.

https://github.com/microsoft/react-native-macos/issues/466